go.mod

```diff
@@ -153,3 +172,5 @@ replace github.com/hashicorp/terraform => github.com/cycloidio/terraform v0.13.5
 
 // Fork of Azurerm that has the V2 of the SDK
 replace github.com/hashicorp/terraform-provider-azurerm => github.com/cycloidio/terraform-provider-azurerm v1.44.1-0.20220324163521-5340b36d0d63
+
+replace github.com/hashicorp/terraform-provider-aws => /opt/go/src/github.com/hashicorp/terraform-provider-aws
```

Comment:
Nop.

Reply:
don't worry, PR is still in progress, I haven't time to push the terraform-provider-aws fork yet ;)